File: pesquisa-e-ordenacao/31-07/java/Main.java
import java.util.ArrayList;
import java.util.Collections;

public class Main {
    public static void main(String[] args) {
        ArrayList<Integer> listaNumeros = new ArrayList<>();
        ArrayList<Integer> listaNumeros2 = new ArrayList<>();
        ArrayList<String> listaPalavras = new ArrayList<>();

        Util.popularAleatorioNumeros(listaNumeros, 10, 1, 100);
        System.out.println("Lista 1: ");
        Util.exibirListaNumeros(listaNumeros);

        listaNumeros2.addAll(listaNumeros);
        Collections.sort(listaNumeros2);
        System.out.println("Lista 2 (ordenada): ");
        Util.exibirListaNumeros(listaNumeros2);

        Util.popularAleatorioPalavras(listaPalavras, 10, 5);
        Util.exibirListaPalavras(listaPalavras);
    }
    

File: pesquisa-e-ordenacao/31-07/python/Main.py
from Util import Util


class Main:
    listaNumeros = []
    listaNumeros2 = []
    listaPalavras = []

    Util.popularAleatorioNumeros(listaNumeros, 10, 1, 100)
    print("Lista 1: ")
    Util.exibirLista(listaNumeros)

    listaNumeros2.extend(listaNumeros)
    listaNumeros2.sort()
    print("Lista 2 (ordenada): ")
    Util.exibirLista(listaNumeros2)

    print("Lista de palavras: ")
    Util.popularAleatorioPalavras(listaPalavras, 10, 5)
    Util.exibirLista(listaPalavras)